geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/StopGatewayReceiverCommand.java

```diff
@@ -69,24 +69,24 @@ public ResultModel stopGatewayReceiver(@CliOption(key = {CliStrings.GROUP, CliSt
         if (receiverBean != null) {
           if (receiverBean.isRunning()) {
             receiverBean.stop();
-            GatewayCommandsUtils.accumulateStartResult(resultData, member.getId(),
+            resultData.addMemberStatusResultRow(member.getId(),
```

Comment:
any reason we can't do things like this:
// to this once in the beginning
resultData.setColumnHeaders ("Member", "Result", "Message")

// then based on the conditions, do this multiple times 
result.addRow(memberName, result, message);